views/widget-front.php

```diff
@@ -44,6 +44,17 @@ class_exists( 'Statify' ) || exit;
 		</table>
 	</div>
 
+	<div class="table searches">
+		<p class="sub"><?php esc_html_e( 'Top searches', 'statify' ); ?></p>
+		<table>
+			<tbody>
+				<?php for ( $i = 0; $i < $limit; $i++ ) { ?>
+					<tr class="placeholder"><td colspan="2">&nbsp;</td></tr>
+				<?php } ?>
```

Comment:
Do I get this right, that you first create a table with empty `<tr>` tags and then fill/replace them using JavaScript? Why not generate the `<table>` in JavaScript instead?

Reply:
OK, you do just the same as with the other tables :see_no_evil: